RFS/src/main/java/com/rfs/models/IndexMetadata.java

```diff
@@ -11,8 +12,24 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.fasterxml.jackson.dataformat.smile.SmileFactory;
+import com.rfs.common.ByteArrayIndexInput;
+import com.rfs.common.RfsException;
+import com.rfs.common.SnapshotRepo;
 
-public class IndexMetadata {
+public interface IndexMetadata extends Index {
+
+    /*
+    * Defines the behavior expected of an object that will surface the metadata of an index stored in a snapshot
+    * See: https://github.com/elastic/elasticsearch/blob/v7.10.2/server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java#L1475
+    * See: https://github.com/elastic/elasticsearch/blob/v6.8.23/server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java#L1284
+    */
+    public JsonNode getAliases();
+    public String getId();
+    public JsonNode getMappings();
```

Comment:
Not a blocker, just trying to understand - it looks like we only updated the 6.8 implementation of this method to return a `JsonNode`, while the 7.10 version still returns an ObjectNode.  Is this intentional or an oversight?

Reply:
Intentional / limiting the number of changes.